### ISSUE TITLE
failed to open configuration file '/usr/local/etc/php/7.0/php-fpm.conf'

### ISSUE THREAD

**Issue by cscd98** [guoxiao/homebrew-php7#4]
brew install php70

/usr/local/opt/php70/sbin/php-fpm --fpm-config /usr/local/etc/php/7.0/php-fpm.conf

```
[02-Jul-2015 11:32:10] ERROR: failed to open configuration file '/usr/local/etc/php/7.0/php-fpm.conf': No such file or directory (2)
[02-Jul-2015 11:32:10] ERROR: failed to load configuration file '/usr/local/etc/php/7.0/php-fpm.conf'
[02-Jul-2015 11:32:10] ERROR: FPM initialization failed
```

**Comment by guoxiao** [comment]
The `config_path` is `/usr/local/etc/php/7`, not `/usr/local/etc/php/7.0`

**Comment by cscd98** [comment]
@guoxiao Thanks. I copied and pasted this for the ~/Library/LaunchAgents/ script. Perhaps that could be updated?

**Comment by guoxiao** [comment]
@craigcarnell I have fixed this bug in 7e95c00.
You can reinstall `php7` and a new plist file will be created
